### PR TITLE
Project Four : Fruit shop

### PR DESCRIPTION
**Pull request by reubengt**
Introduced terminal git usage here as well

### CODE REVIEW

File: src/course/syllabus/application/four/learning-outcomes.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+
+## DOM
+
+- [ ] Be able to create and update elements on the page that depend on an object for content
+- [ ] Be able to re-use an event listener callback function for different buttons (eg adding to cart)
+
+## Terminal/Command Line
+
+- [ ] Know how to use the terminal to navigate through a file system
+- [ ] Know how to create a directory or create a file
```
Comment (Albadylic) [inline src/course/syllabus/application/four/learning-outcomes.md:33]:
```suggestion
- [ ] Know how to create a directory or file
```

File: src/course/syllabus/application/four/project.md [hunk 2]
```diff
@@ -1 +1,19 @@
-project
+# Fruit Shop
```
Comment (Albadylic) [inline src/course/syllabus/application/four/project.md:1]: I love this project ✨

**Comment by oliverjam** [comment]
Yeah we probably need a full curriculum issue to discuss what we want to recommend going forward for Windows. WSL2 is kind of complex for a complete beginner to set up, and seems to have a lot of rough edges down the line. We could try and get away with telling them to use Git Bash or something and deal with the slight differences in environment.

File: src/course/syllabus/application/four/project.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+
+Your Project this week is to build a fruit shop app that lists produce and allows a user to create a shopping basket!
+
+{% box %}
+For this project, we also expect you to research how to use git from the command line, instead of using the source control tab on VS code.
+
+If you have not used the terminal/command line before - it's also a good idea to do some research on navigating your file system using the terminal, and using commands like `cd`, `ls`and `touch`/`echo`
+{% endbox %}
```
Comment (oliverjam) [inline src/course/syllabus/application/four/project.md:9]: Small markdown/shortcodes thing: if you want the content inside a shortcode to be treated as markdown it needs to have linebreaks around it. E.g.
```md
{% box %}
_hello_
{% endbox %}
```
will produce
```html
<div class="box">
_hello_
</div>
```
whereas
```md
{% box %}

_hello_

{% box %}
```
will produce
```html
<div class="box">
<p><em>hello</em></p>
</div>
```

**Pull request merged**